packages/cli/package.json

```diff
@@ -2,9 +2,16 @@
   "name": "@pipcook/pipcook-cli",
   "version": "0.5.29",
   "description": "",
-  "main": "index.js",
+  "main": "dist/bin/index.js",
+  "types": "dist/bin/index.js",
```

Comment:
What's this for?

Reply:
Entrance of cli.

Reply:
See https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package, the types property in package.json should be TypeScript definition file, this is invalid for that.

Reply:
removed